api/cache/redis.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import createRedis from 'shared/bull/create-redis';
```

Comment:
Instead of importing this and then not using anything from that file, let's do this instead:

```JS
import Redis from 'ioredis';

/* ... */

const createRedis = (extra) => new Redis({ ...config, ...extra });

export default createRedis;
```

Way less confusing!